### PR TITLE
Julia's spotify.me homework

### PR DESCRIPTION
**Pull request by julia-rose**
:sound: :headphones: :microphone: :guitar: :trumpet: :saxophone:

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
What are you doing here? You already graduated!

**Comment by julia-rose** [comment]
@RobertAKARobin I'm in the new JS development class (with Jesse)! Tryna' step my game up! :grin:

File: js/spotify.js [hunk 1]
```diff
@@ -1,12 +1,125 @@
 // API Docs at: 
 // https://developer.spotify.com/technologies/web-api/search/
 
+'use strict';
 
-function searchByArtist(keyword) {
-  var url = 'http://ws.spotify.com/search/1/artist.json?q='+keyword;
```
Comment (recursivefunk) [inline js/spotify.js:6]: Kill `var` everywhere. You don't have a need for it anymore. [Read this](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#.a4shy0x86)

File: js/spotify.js [hunk 2]
```diff
@@ -1,12 +1,125 @@
...
+  } else {
+    composeQuery(keyword, 'track');
+  }
+};
+
+// Using search term and search type, compose the Spotify endpoint URL
+const composeQuery = (keyword, searchType) => {
+  var url = 'https://api.spotify.com/v1/search?q=' + keyword + '&type=' + searchType;
+
+  searchSpotify(url);
```
Comment (recursivefunk) [inline js/spotify.js:31]: Love the use of a `composeQuery` function. I think you have the right idea here. I would not include the `searchSpotify()` function call here though. We want our functions to focus on one task. By having `searchSpotify()` function call, we _assume_ that anytime we compose a query that we **immediately** want to search spotify which may or may not be the case!

**Comment by recursivefunk** [comment]
Excellent job! Made a couple comments

**Pull request closed without merging**